spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/negotiation/ContractRequestData.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright (c) 2021 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.contract.spi.types.negotiation;
+
+import org.eclipse.edc.connector.contract.spi.types.offer.ContractOffer;
+
+import java.util.Objects;
+
+/**
+ * Represent the data needed for a contract negotiation request
```

Comment:
Should the javadoc say to initiate a contract request on the consumer side?

Reply:
:+1: